% Template for PLoS
% Version 3.1 February 2015

\documentclass[10pt,letterpaper]{article}
\usepackage[top=0.85in,left=2.75in,footskip=0.75in]{geometry}

% Use adjustwidth environment to exceed column width (see example table in text)
\usepackage{changepage}

% Use Unicode characters when possible
\usepackage[utf8]{inputenc}

% textcomp package and marvosym package for additional characters
\usepackage{textcomp,marvosym}

% fixltx2e package for \textsubscript
\usepackage{fixltx2e}

% amsmath and amssymb packages, useful for mathematical formulas and symbols
\usepackage{amsmath,amssymb}

% cite package, to clean up citations in the main text. Do not remove.
\usepackage{cite}

% Use nameref to cite supporting information files (see Supporting Information section for more info)
\usepackage{nameref,hyperref}

% line numbers
\usepackage[right]{lineno}

% ligatures disabled
\usepackage{microtype}
\DisableLigatures[f]{encoding = *, family = * }

% rotating package for sideways tables
\usepackage{rotating}

% Remove comment for double spacing
\usepackage{setspace} 
%\doublespacing

% Text layout
\raggedright
\setlength{\parindent}{0.5cm}
\textwidth 5.25in 
\textheight 8.75in

% Bold the 'Figure #' in the caption and separate it from the title/caption with a period
% Captions will be left justified
\usepackage[aboveskip=1pt,labelfont=bf,labelsep=period,justification=raggedright,singlelinecheck=off]{caption}

% Use the PLoS provided BiBTeX style
\bibliographystyle{plos2015}

% Remove brackets from numbering in List of References
\makeatletter
\renewcommand{\@biblabel}[1]{\quad#1.}
\makeatother

% Leave date blank
\date{}

% Header and Footer with logo
\usepackage{lastpage,fancyhdr,graphicx}
\usepackage{epstopdf}
\pagestyle{myheadings}
\pagestyle{fancy}
\fancyhf{}
\lhead{\includegraphics[width=2.0in]{PLOS-submission.eps}}
\rfoot{\thepage/\pageref{LastPage}}
\renewcommand{\footrule}{\hrule height 2pt \vspace{2mm}}
\fancyheadoffset[L]{2.25in}
\fancyfootoffset[L]{2.25in}
\lfoot{\sf PLOS}
\rhead{Purslane \& Saline Stress} % Short Title

%% Include all macros below

\newcommand{\lorem}{{\bf LOREM}}
\newcommand{\ipsum}{{\bf IPSUM}}

%% END MACROS SECTION

%% Next items added locally, not part of the PLOS template
%\usepackage[nomarkers, nolists, figuresonly]{endfloat}

\usepackage{booktabs}
\setlength\heavyrulewidth{2 pt}
\setlength\lightrulewidth{1 pt}

\usepackage{float}
\floatstyle{plaintop}
\restylefloat{table}

% Now the document

\begin{document}
\vspace*{0.35in}

\begin{flushleft}
\Large{
\textbf{Alternative Strategies in Response to Saline Stress in Two Varieties of \emph{Portulaca oleracea} (Purslane)}
}\\

Kristina R. Mulry\textsuperscript{1,\textcurrency},
Bryan A. Hanson\textsuperscript{1, *},
Dana A. Dudle\textsuperscript{2}
\\
\bigskip
\textbf{1} Dept. of Chemistry \& Biochemistry, DePauw University, Greencastle, Indiana, USA
\\
\textbf{2}  Dept. of Biology, DePauw University, Greencastle, Indiana, USA
\\
\bigskip

\textcurrency ~Indiana University School of Medicine, Indianapolis, Indiana, USA

* Corresponding author.  E-mail: hanson@depauw.edu

\end{flushleft}

\section*{Abstract}

Purslane (\emph{Portulaca oleracea}) is a globally-distributed plant with a long history of use in folk medicine and cooking. We have developed purslane as a model system for exploring plant responses to stress. We exposed two varieties of purslane to saline stress with the objective of identifying differences between the varieties in the plasticity of morphological and physiological traits. The varieties responded to saline stress with significantly different changes in the measured traits, which included \emph{inter alia} biomass, flower counts, proline concentrations and betalain pigment concentrations. The alternative responses of the two varieties consisted of complex, simultaneous changes in multiple traits. In particular, we observed that while both varieties increased production of betalain pigments and proline under saline stress, one variety invested more in betalain pigments while the other invested more in proline. Proline and betalain pigments undoubtedly play multiple roles in plant tissues, but in this case their role as antioxidants deployed to ameliorate saline stress appears to be important. Taken holistically, our results suggest that the two varieties employ different strategies in allocating resources to cope with saline stress. This conclusion establishes purslane as a suitable model system for the study of saline stress and the molecular basis for differential responses.

%\linenumbers

\section*{Introduction}

Purslane (\emph{Portulaca oleracea}) is succulent, robust medicinal plant, distributed in temperate and tropical regions world-wide. It thrives not only in gardens and cultivated fields, but in waste spaces and sidewalk cracks.  It has been used extensively in folk medicine since antiquity, and also as a nutritious pot herb and salad ingredient.\cite{Mitich1997, Holm1991, Uddin2014}

Purslane has been investigated extensively for bioactive compounds.\cite{Zhou2015}  The most noteworthy are the $\omega$-3 fatty acids, and in particular, $\alpha$-linolenic acid (ALA). This class of compounds and their metabolites have been identified as inhibitors of tumorigenesis \cite{Murray2014} and promoters of good cardiovascular health.\cite{Riediger2009} Among terrestrial sources, purslane has the highest amount of ALA, with approximately 4 mg ALA per gram of fresh plant.\cite{Simopoulos1992}  Purslane is also rich in antioxidant compounds and it activates enzymes that quench oxidants.\cite{Alam2014a, Erkan2012, Dkhil2011} Like most members of the order Caryophylallales, purslane synthesizes betalain pigments rather than anthocyanins.\cite{Ocampo2012, Brockington2011} These pigments are antioxidants and of great interest as natural food colorants and nutritional supplements, \cite{Stintzing2007} and have been shown to protect low density lipoprotein particles.\cite{Allegra2007} Polysaccharides are abundant as well, and exhibit antiviral, antidiabetic and antitumor activities.\cite{Zhao2013, Dong2010, Shen2013} The seeds\cite{Esmailzadeh2015} and aqueous extracts\cite{Lee2012b, Lee2012c} have been found to be useful in the control of diabetes and to ameliorate some side effects of the disease.  Finally, purslane extracts have shown neurological effects worthy of further investigation.\cite{Wang2010, Moneim2013}

The high level of ALA has led researchers to investigate the conditions necessary to optimize the production of fatty acids.  The influence of nitrogen source\cite{Fontana2006}, stage of harvest\cite{Mortley2012} and  light\cite{Palaniswamy2001a} have been reported.  Purslane has also been studied for the remediation of soils and industrial effluents polluted with metals\cite{Tiwari2008}, salt\cite{Kilic2008} and endocrine disruptors like bisphenol A.\cite{Imai2007} An interesting characteristic of the genus \emph{Portulaca} is that it contains the only known example of a C-4 plant which can switch to crassulacean acid metabolism under drought stress, and \emph{P. oleracea} in particular does not have photosynthetic stems.\cite{Ocampo2012, Winter2014, DAndrea2014}

We have identified purslane as a suitable model system for exploring plant responses to stress, particularly saline stress.  Recent modeling of anthropogenic climate change predicts greater extremes in temperature and rainfall, with a net increase in global temperature and a decrease in rainfall.  These changes are not expected to be evenly distributed, and it is anticipated that semi-arid regions will expand significantly.\cite{IPCC2014a}  Higher temperatures in agricultural regions will lead to increased irrigation, which in turn will increase soil salinity in an unpredictable pattern across the range of widespread species such as purslane. At the same time, evidence is mounting that plant species ranges and community assemblages are already responding to climate change in complex ways.\cite{Corlett2013, Franks2014} 

Because plants are sessile, individuals that exhibit phenotypic plasticity in response to environmental stressors may buffer the effects of unpredictably harsh environments resulting from climate change. Such plasticity may allow some individuals to colonize hostile sites and reproduce despite saline conditions.\cite{Sultan2003} Adaptive plasticity in functional traits such as leaf mass and presence and concentration of secondary metabolites has been identified as a key factor in the effectiveness of plants' response to unpredictable salinity within a generation. \cite{Nicotra2010, Sultan2003} 

Over longer time frames, natural selection should favor plants with greater ability to cope with saline soils and other stressors as the climate changes more permanently.  These genotypes must have functionally integrated biochemical phenotypes that improve their fitness and survival.\cite{Ahuja2010}  At the scale of plant populations and communities, extirpation is a real possibility if species cannot adapt quickly, disperse into suitable habitats or find refugia. Thus, climate change not only threatens crops and the food supply, but also biodiversity and the accompanying ecosystem services. 

We report here our investigations into the responses of two varieties of purslane to saline stress.  In particular, we investigated whether both varieties exhibit the same level of plasticity in response to saline stress, and whether functional traits such as biomass, flower production, and concentration of secondary metabolites respond to saline stress in a coordinated fashion, using ecological and chemical methods.  Because purslane is widespread and synthesizes a wide array of bioactive compounds that are likely associated with adaptation to stress, it is possible that different varieties may employ alternative strategies when faced with similar stressors. Studying the stress response using multiple functional traits may reveal important clues about complex adaptive strategies plants employ to cope with saline stress induced by climate change.

\section*{Plant Materials and Methods}

\emph{Experimental design}.  To examine variation in response to saline stress between cultivated and wild varieties of purslane, two isogenic varieties of purslane (T-16 and WI-9) were grown in two saline treatments (0 or 200 mM NaCl). T-16 was derived from the seeds of single plant of the cultivated variety 'Tall Green' (obtained from Wild Garden Seed, www.wildgardenseed.com in 2009). WI-9 is a wild variety derived from seeds of a single plant collected in Madison, Wisconsin in 2009. Both varieties were maintained locally via self-fertilization for several generations. Twenty plants (replicates) of each variety were assigned to each of the two saline treatments (the actual number of plants in each analysis varies and is slightly less than 20 due to sample loss) .

\emph{Seedling establishment, saline treatments, and harvest}.  Five to 10 seeds of \emph{P. oleracea} were sown on the surface of MetroMix 360 potting soil in 3.5 inch diameter plastic pots.  The pots were placed in an insect-free greenhouse, and their position randomized with respect to their variety and assigned treatment.  Ample water was provided to all pots during the two-week seedling establishment period.  At the end of the establishment period, plants were thinned to one plant per pot before saline treatments began. After the plants were thinned, the number of nodes on the main stem and the number of branches from the main stem were recorded, to verify that no significant pre-treatment differences existed among plants of the same variety assigned to the two treatments.  We applied one of two treatments to each pot:  0 mM NaCl or 200 mM NaCl.  The 0 mM NaCl pots were watered three times per week with 56 mL tap water.  The 200 mM NaCl pots were watered three times per week with 56 mL of a 200 mM NaCl solution prepared in tap water. These saline treatments were chosen based on results from preliminary experiments (data not shown). 
  
\emph{Harvest and collection of morphological trait data}.  After 10 treatments over 21 days (35 days from sowing), we recorded the number of nodes on the main stem, the number of axillary branches produced on the main stem, the number of leaves dropped from the main stem, and the number of flower structures produced by each plant. Several leaves, which emerged during the treatment period, and a small segment of stem were harvested for chemical analysis.  The above-ground biomass was determined by cutting the stem at soil level, and drying the plant material in an 80 $^{\circ}$C oven for 3-5 days.  All masses were determined to 0.1 mg on an analytical balance.

\emph{Chemical analysis}. Proline levels were measured using the method of Bates\emph{et al.}.\cite{Bates1973}  Briefly, pieces of fresh leaves from each plant were sliced into small pieces and placed in 3\% sulfosalicylic acid solution, then shaken for 60 s in a Bead Beater (no beads were added).  The supernatant was mixed with ninhydrin solution and glacial acetic acid, then heated at 100 $^{\circ}$C for 1 h.  After extraction into toluene, the absorbance at 520 nm was recorded.  Comparison was made to a calibration curve prepared with pure proline. Results are reported in $\mu$g proline/g fresh leaf.

Betalain pigment levels were measured using the method of Kugler \emph{et al.}.\cite{Kugler2004} Briefly, pieces of plant tissue were placed in 60\% methanol along with two steel beads and shaken for 60 s in a Bead Beater.  We recorded the absorbance of the supernatant across the visible spectrum using an Ocean Optics spectrometer.  The indices developed by Elbe\cite{Elbe2007} were used to compute the amounts of each pigment.  Results are reported in $\mu$g pigment/g fresh tissue.

Total phenolic compounds were measured using the bicinchoninic acid (BCA) assay.\cite{Marino2009} Briefly, pieces of fresh leaves were placed in 60\% methanol along with two steel beads and shaken for 60 s in a Bead Beater.  The supernatant was mixed with ammonium acetate buffer and Cu(II) solution.  BCA solution was added.  The absorbance at 558 nm was recorded and compared to a calibration curve prepared with gallic acid.    Results are reported in $\mu$g GAE/g fresh leaf (GAE = gallic acid equivalents).

\emph{Statistics} Statistical analysis was carried out using R 3.3\cite{R3.3} and the R package HandyStuff was used to draw the reaction norms.\cite{Handy2015}  Prior to conducting ANOVA, variables were examined using a normal probability plot (QQ plot) to assess whether the variables appeared to arise from a normal distribution.  While some variables had outliers, no non-normal trend was evident and the data were not transformed.  A result was considered significant if the p-value was $\le$ 0.05.

<<  setUp, include = FALSE >>=

rm(list = ls()) # clean the workspace to avoid cross-talk

suppressMessages(library("knitr"))
suppressMessages(library("HandyStuff")) # github.com/bryanhanson/HandyStuff
suppressMessages(library("reshape2"))
suppressMessages(library("ggplot2"))
suppressMessages(library("plyr"))
suppressMessages(library("gridExtra"))
suppressMessages(library("lattice"))
suppressMessages(library("latticeExtra"))
suppressMessages(library("ChemoSpec"))
suppressMessages(library("grid"))
suppressMessages(library("jpeg"))
suppressMessages(library("broom"))
suppressMessages(library("xtable"))

# include = FALSE means don't put anything in the report (but the calc is done and figure is made)

# This next chunk is used for PLOS ONE.  Suppresed here to avoid the tiff files
#opts_chunk$set(tidy = TRUE, cache = FALSE, echo = FALSE, include = FALSE,
#	fig.path = 'figures/', dev = c("pdf", "tiff"), dpi = c(300, 300),
#	dev.args = list(tiff = list(compression = "lzw")),
#	fig.width = 7, fig.height = 7) # default figure size is 7 x 7 inches

opts_chunk$set(tidy = TRUE, cache = FALSE, echo = FALSE, include = FALSE,
	fig.path = 'figures/', dev = c("pdf"), dpi = c(300),
	fig.width = 7, fig.height = 7) # default figure size is 7 x 7 inches

# local variables for use with rxnNorm

rNcol <- c("darkgreen", "red") # colors for rxnNorm plots (tall GREEN/Wisconsin RED)
rNcex <- 0.75 # table text size in plots
meth <- "sem95" # method for error bars
xlabspc <- "Concentration of NaCl"
source("screenThemePole1.R")

# Function to report figure captions independent of figures, as required by PLOS One.
# Original idea from Yihui Xie but modified considerably.

x <- NULL
ii <- 0 # Current figure number
fn <- function(x) {
    ii <<- ii + 1
    figString <<- paste('\\textbf{Figure ', ii, '.} ', x, sep = '') # Current caption saved in global environment
    return(x) # Return original fig.cap so it can be processed normally in the LaTeX figure environment
  }

set.seed(7) # Needed so that jittering is consistent from run-to-run
@

<<  GetData >>=
#
# Bring in the master data set
#
po <- read.csv("PO_12July2013.csv")
names(po)[2] <- "variety"
@

<<  CalcMasses >>=
#
# The following variables relating to masses will be computed once here
# and used in various places later. All masses in grams unless otherwise noted.
#
po$d2w <- ((po$AGdry.mass.7.8 - po$env.7.8)/(po$AGwt.mass.7.1 - po$env.7.8))
po$netDryMass <- po$AGdry.mass.7.8 - po$env.7.8
po$netFreshMass <- po$AGwt.mass.7.1 - po$env.7.8
@

\section*{Results}

Both varieties of purslane responded to saline stress with significant changes in morphological and physiological traits including a reduced investment in reproductive structures (Fig.~\ref{morphFotos}).  However, the two varieties varied significantly in their specific responses to the saline environment, demonstrating that these varieties of \emph{Portulaca oleracea} employ alternative strategies to cope with this type of stress.

%%%%%  Figure 1 (Photos)

<< Fig_1, fig.cap = fn("\\label{morphFotos}\\textbf{A: Variety T-16; B: Variety WI-9}.  Plants on the left were treated with 0 mM NaCl while plants on the right were treated with 200 mM NaCl."), include = TRUE >>=
dvp1 <- viewport(x = 0.25, y = 0.5, width = 0.5, height = 0.5) # upper
dvp2 <- viewport(x = 0.65, y = 0.375, width = 0.5, height = 0.25) # lower

grid.newpage()
pushViewport(dvp1)
grid.raster(readJPEG("photos/T-16_SM.jpg"))
popViewport()
pushViewport(dvp2)
grid.raster(readJPEG("photos/WI-9_SM.jpg"))
popViewport()

grid.text("A", x = 0.1, y = 0.78, gp = gpar(fontface = "bold", cex = 1.5))
grid.text("B", x = 0.47, y = 0.53, gp = gpar(fontface = "bold", cex = 1.5))
@

% Setting include = TRUE on this next code chunk will place the figure
% caption in the correct spot as required in a PLOS ONE submission.
% However, include = FALSE here so that the figures will be placed
% by LaTeX.  When include = TRUE, you also need to use endfloats
% in the preamble.
<< Fig_1_Caption, include = FALSE, results = "asis" >>=
cat(figString)
@

\subsection*{Morphological Traits}

At the end of the 14-day establishment period, ANOVA was used to verify that the seedlings assigned to each treatment were similar within a variety, prior to the actual start of treatments (data not shown).  Fig~\ref{morphAll} shows the variation in morphological traits with saline stress after 21 days of treatment.  There were significant variety by treatment interactions for all variables (Table~\ref{Table1}).  Both varieties produced fewer flowers and less dry biomass under saline stress.   The decrease in flowering structures was particularly dramatic for variety WI-9.  Variety T-16  produced fewer nodes and dropped more leaves from the main stem under the saline treatment than variety WI-9.  However, variety WI-9 showed little change for these two variables under saline stress (see also \nameref{S1_Fig}, \nameref{S2_Fig} and \nameref{S3_Fig}).

%%%%%   Prepare Figures 2a - 2d

<<  Fig_2a  >>=
ylabspc <-  "flower count"
Fig2a <- rxnNorm(formula = flrs7.1 ~ treatment, groups = variety, type = "connect", data = po, freckles = TRUE, method = meth, cols = rNcol, xlab = "",  ylab = ylabspc, ylim=c(-1, 28), theme = screenThemeSPC(), legend = NULL)
@

<<  Fig_2b >>=
ylabspc <- "node count"
Fig2b <- rxnNorm(formula = nodes7.1 ~ treatment, groups = variety, type = "connect", data = po, freckles = TRUE, method = meth, cols = rNcol, xlab = "", ylab = ylabspc, ylim=c(-1, 14), theme = screenThemeSPC(), legend = NULL)
@

<<  Fig_2c  >>=
ylabspc <-"dropped leaf count"
Fig2c <- rxnNorm(formula = drplvs7.1 ~ treatment, groups = variety, type = "connect", data = po, freckles = TRUE, method = meth, cols = rNcol, xlab = "", ylab = ylabspc, ylim = c(-0.5, 5.2), theme = screenThemeSPC(), legend = NULL)
@

<<  Fig_2d >>=
ylabspc <- "above-ground dry mass (g)"
Fig2d <- rxnNorm(formula = netDryMass ~ treatment, groups = variety, data = po, freckles = TRUE, method = meth, xlab = "", ylab = ylabspc, type = "connect", cols = rNcol, ylim= c(-0.05, 1.8), theme = screenThemeSPC(), legend = NULL)
@

%%%%%   Figure 2 (2a - 2b in 4 x 4 panel - Morphological Data)

<< Fig_2, fig.cap =  fn("\\label{morphAll}\\textbf{Morphological traits at 21 days.} Variety T-16 shown in green.  Variety WI-9 shown in red.  Large point is the mean of the category.  Small points are the sample values, jittered horizontally.  Error bars correspond to the 95\\% confidence interval."), include = TRUE  >>=
dvp1 <- viewport(x = 0.25, y = 0.75, width = 0.5, height = 0.5) # upper left
dvp2 <- viewport(x = 0.25, y = 0.25, width = 0.5, height = 0.5) # lower left
dvp3 <- viewport(x = 0.75, y = 0.75, width = 0.5, height = 0.5) # upper right
dvp4 <- viewport(x = 0.75, y = 0.25, width = 0.5, height = 0.5) # lower right

grid.newpage()
pushViewport(dvp1)
print(Fig2a, newpage = FALSE)
popViewport()
pushViewport(dvp2)
print(Fig2b, newpage = FALSE)
popViewport()
pushViewport(dvp3)
print(Fig2c, newpage = FALSE)
popViewport()
pushViewport(dvp4)
print(Fig2d, newpage = FALSE)
popViewport()

grid.text("A", x = 0.05, y = 0.95, gp = gpar(fontface = "bold", cex = 1.5))
grid.text("B", x = 0.05, y = 0.45, gp = gpar(fontface = "bold", cex = 1.5))
grid.text("C", x = 0.55, y = 0.95, gp = gpar(fontface = "bold", cex = 1.5))
grid.text("D", x = 0.55, y = 0.45, gp = gpar(fontface = "bold", cex = 1.5))
@

% Setting include = TRUE on this next code chunk will place the figure
% caption in the correct spot as required in a PLOS ONE submission.
% However, include = FALSE here so that the figures will be placed
% by LaTeX.  When include = TRUE, you also need to use endfloats
% in the preamble.
<< Fig_2_Caption, include = FALSE, results = "asis" >>=
cat(figString)
@

%%%%%  Table 1

\vspace{0.5 cm}

<< Table1, include = TRUE, results = "asis" >>=

# run the anovas
aov1 <- tidy(aov(flrs7.1 ~ treatment*variety, data = po))
aov2 <- tidy(aov(nodes7.1 ~ treatment*variety, data = po))
aov3 <- tidy(aov(drplvs7.1 ~ treatment*variety, data = po))
aov4 <- tidy(aov(netDryMass ~ treatment*variety, data = po))

# assemble
bigaov <- rbind(aov1, aov2, aov3, aov4)
bigaov$df <- as.integer(bigaov$df)
bigaov <- cbind(
	c("\\textbf{flowers}", rep("", 3), "\\textbf{nodes}", rep("", 3), "\\textbf{dropped}",
	"\\textbf{leaves}", rep("", 2), "\\textbf{dry}", "\\textbf{mass}", rep("", 2)), bigaov)
names(bigaov)<- c("\\textbf{trait}", "\\textbf{term}", "\\textbf{df}", "\\textbf{SS}",
	"\\textbf{MS}", "\\textbf{F-statistic}", "\\textbf{p-value}")

# format for output
table1 <- xtable(bigaov, label = "Table1", caption = "\\textbf{ANOVA results for morphological traits.}")
align(table1) <- "rrrrrrrr"
digits(table1)[8] <- -2
digits(table1)[4:7] <- 1
print(table1, floating = TRUE, include.rownames = FALSE,
	hline.after = c(-1, 0, 4, 8, 12, 16), booktabs = TRUE,
	sanitize.text.function = function(x){x},
	scalebox = 0.85, table.placement = "H",
	latex.environment = "")
@

\vspace{0.5 cm}

\subsection*{Physiological Traits}

The BCA assay was conducted as a measure of the total concentration of phenolic compounds which can serve as antioxidants. The results indicated a modest variety by treatment interaction, but no statistically significant main effects (Fig.~\ref{physioAll} Panel A and Table~\ref{Table2}). The total phenols (measured as GAE or gallic acid equivalents) of variety T-16 decreased about 25\% with increased salinity, but variety WI-9 remained largely unchanged. The concentration of the amino acid proline after 21 days of treatment exhibited a significant variety by treatment interaction (Fig.~\ref{physioAll} Panel B and Table~\ref{Table2}).  Variety T-16 responded to the saline stress by producing about 75\% more proline than variety WI-9, per gram of fresh mass.

<<  BCAcalCurveSpectra >>=
BCA_cc <- read.csv("BCA/BCA_Cal_Curve.csv")
BCA_cc <- BCA_cc[,-2] # remove spectra that drop below 0
BCA_cc <- BCA_cc[,-3]
BCA_cc.m <- melt(data = BCA_cc, id.vars = "Wavelength")
#
# The following is not part of the paper, but can be uncommented to inspect
# the data/results (the figure will not be included in the paper, but will be in the figures folder)
#
#KM1 <- qplot(data = BCA_cc.m, x = Wavelength, y = value, geom = "line", color = variable, ylab = "Absorbance", main = #	"BCA Calibration Spectra (21 Days)") + theme(legend.position = "none") + geom_vline(xintercept = 558)
#print(KM1)
@

<<  BCAcalCurve >>=
BCA_lm <- data.frame(absorbance = colMeans(BCA_cc[c(487:492),-1]), concentration = c(1.13, 2.26, 3.76))
#
# The following is not part of the paper, but can be uncommented to inspect
# the data/results (the figure will not be included in the paper, but will be in the figures folder)
#
#KM2 <- lmEqn(formula = absorbance ~ concentration, data =  BCA_lm,
#	ylab = "absorbance", xlab = "[GAE], ppm", leg.loc = c(1, 0.7), main = "BCA Calibration Curve (21 Days)")
#print(KM2)
@

<<  BCAsampleSpectra >>=
BCA <- read.csv("BCA/BCA_Samples.csv")
#
# The following is not part of the paper, but can be uncommented to inspect
# the data/results (the figure will not be included in the paper, but will be in the figures folder)
#
#BCA.m <- melt(data = BCA, id.vars = "Wavelength")
#KM3 <- qplot(data = BCA.m, x = Wavelength, y = value, geom = "line", color = variable, ylab= "Absorbance", xlab = #	"wavelength (nm)", main = "BCA Sample Spectra at 21 Days") + theme(legend.position = "none") + geom_vline(xintercept = 558)
#print(KM3)
@

<<  BCAsamples >>=
A558 <- colMeans(BCA[c(487:492),-1]) # take mean over range of wavelengths
A558 <- data.frame(A558 = unlist(A558), spc = names(A558))
po2 <- join(po, A558, by = "spc", type = "left")

fit <- lm(absorbance ~ concentration, data = BCA_lm)
po2$ppmGAE <-  (po2$A558 - fit$coef[1])/fit$coef[2]
# 100 uL of 1500 uL went into the assay/see notes
# convert to ug (continue for now to call it the same thing...)
po2$ppmGAE <- po2$ppmGAE*1500*1000/(100*po2$BCA.beta.lf.wt.mass.7.1)
@

%%%%% Figure 3a (GAE from BCA assay)

<<  Fig_3a >>=
ylabspc <- expression(GAE/fresh~leaf~(mu*g/g))
Fig3a <- rxnNorm(formula = ppmGAE ~ treatment, groups = variety, type = "connect", data = po2, freckles = TRUE, method = meth, cols = rNcol, ylab = ylabspc, theme = screenThemeSPC(), legend = NULL, xlab = "")
@

<<  ProlineCalCurveSpectra >>=
Bates_cc <- read.csv("Proline/Proline_Cal_Curve.csv")
#
# The following is not part of the paper, but can be uncommented to inspect
# the data/results (the figure will not be included in the paper, but will be in the figures folder)
#
#Bates_cc.m <- melt(data = Bates_cc, id.vars = "Wavelength")
#KM6 <- qplot(data = Bates_cc.m, x = Wavelength, y = value, geom = "line", color = variable, main = "Proline Calibration #	Spectra (21 Days)", ylab = "Absorbance") + geom_vline(xintercept = 520) + theme(legend.position = "none")
#print(KM6)
@

<<  ProlineCalCurve >>=
Bates_lm <- data.frame(absorbance = colMeans(Bates_cc[c(379:385),-1]), concentration = c(0.625, 10.0, 2.25, 20, 40, 5))
#
# The following is not part of the paper, but can be uncommented to inspect
# the data/results (the figure will not be included in the paper, but will be in the figures folder)
#
#KM7 <- lmEqn(formula = absorbance ~ concentration, data =  Bates_lm,
#   ylab = "absorbance", xlab = "ug proline/mL", leg.loc = c(0.0, 1.2), main = "Proline Calibration Curve (21 Days)")
#print(KM7)
@

<<  ProlineSampleSpectra >>=
bates <- read.csv("Proline/Proline_Samples.csv")
#
# The following is not part of the paper, but can be uncommented to inspect
# the data/results (the figure will not be included in the paper, but will be in the figures folder)
#
#bates.m <- melt(data = bates, id.vars = "Wavelength")
#KM8 <- qplot(data = bates.m, x = Wavelength, y = value, ylab = "Absorbance", geom = "line", color = variable, main = #	"Proline Sample Spectra at 21 Days") + geom_vline(xintercept = 520) + theme(legend.position = "none")
#print(KM8)
@

<<  ProlineSamplesA >>=
A520 <- colMeans(bates[c(379:385),-1]) #take mean over range of wavelengths
A520 <- data.frame(A520 = unlist(A520), spc = names(A520))
po2 <- join(po2, A520, by = "spc", type = "left")

fit <- lm(absorbance ~ concentration, data = Bates_lm)
po2$proline <- (po2$A520 - fit$coef[1])/fit$coef[2]
# see assay for details of next step; also, masses here are in mg!
po2$proline <- po2$proline* 4 *  1000/(po2$Bates.lf.wt.mass.7.1)
@

%%%%% Figure 3b (Proline)

<<  Fig_3b  >>=
ylabspc <- expression(proline/fresh~leaf~(mu*g/g))
Fig3b <- rxnNorm(formula = proline ~ treatment, groups = variety, type = "connect", data = po2, freckles = TRUE, method = meth, cols = rNcol, ylab = ylabspc, theme = screenThemeSPC(), legend = NULL, xlab = "")
@


<<  StemBetSpectra >>=
betastem <- read.csv("Betalains/Betalain_Stem_Samples.csv")
#betastem.m <- melt(data = betastem, id.vars = "Wavelength")
#KM14 <- qplot(data = betastem.m, x = Wavelength, y = value, geom = "line", color = variable, main = "Betalain Stem #Spectra at 21 Days", xlab = "Wavelength (nm)", ylab = "Absorbance") + geom_vline(xintercept = 538) + #theme(legend.position= "none")
#print(KM14)
@

<<  StemBetIndices >>=
A538STEM <- colMeans(betastem[c(431:437),-1])
A600STEM <- colMeans(betastem[c(606:612),-1])
AbetacyaninsSTEM <- 1.095*(A538STEM-A600STEM) # using the indices
impuritiesSTEM <- A538STEM - AbetacyaninsSTEM
A476STEM <- colMeans(betastem[c(259:265),-1])
AbetaxanthinsSTEM <- A476STEM - impuritiesSTEM - (AbetacyaninsSTEM/3.1)

# take into account the assay volumes & unit conversions
cyanSTEM <- AbetacyaninsSTEM * 3 * 1.5 * 0.01 * 10^6/1120
xanthSTEM <- AbetaxanthinsSTEM * 3 * 1.5 * 0.01 * 10^6/750

# add to master data frame
cyanSTEM <- data.frame(cyanSTEM = unlist(cyanSTEM), spc = names(cyanSTEM))
po2 <- join(po2, cyanSTEM, by = "spc", type = "left")
xanthSTEM <- data.frame(xanthSTEM = unlist(xanthSTEM), spc = names(xanthSTEM))
po2 <- join(po2, xanthSTEM, by = "spc", type = "left")
@

<<  StemBetaxanA>>=
po2$ug.xanth.frsh.stem <- 1000*po2$xanthSTEM/(po2$beta.stem.wt.mass.7.1)
@

<<  LeafBetSpectra >>=
betaleaf <- read.csv("Betalains/Betalain_Leaf_Samples.csv")
#
# The following is not part of the paper, but can be uncommented to inspect
# the data/results (the figure will not be included in the paper, but will be in the figures folder)
#
#betaleaf.m <- melt(data = betaleaf, id.vars = "Wavelength")
#KM19 <- qplot(data = betaleaf.m, x = Wavelength, y = value, geom = "line", color = variable, main = "Leaf Betalain
#	Spectra at 21 Days", ylab = "Absorbance") + geom_vline(xintercept = 538) + theme(legend.position="none")
@

%%%%% Figure 3c (Betaxanthins/Stem)

<<  Fig_3c >>=
ylabspc <- expression(betaxanthins/fresh~stem~(mu*g/g))
Fig3c<- rxnNorm(formula = ug.xanth.frsh.stem ~ treatment, groups = variety, type = "connect", data = po2, freckles = TRUE, method = meth, cols = rNcol, ylab = ylabspc, theme = screenThemeSPC(), legend = NULL, xlab = "")
@

<<  StemBetacyanA >>=
po2$ug.cyan.frsh.stem <- 1000*po2$cyanSTEM/(po2$beta.stem.wt.mass.7.1)
po2$ug.total.betalains.stem <- po2$ug.cyan.frsh.stem + po2$ug.xanth.frsh.stem
@

%%%%% Figure 3d (Betacyanins/Stem)

<<  Fig_3d >>=
ylabspc <- expression(betacyanins/fresh~stem~(mu*g/g))
Fig3d <- rxnNorm(formula = ug.cyan.frsh.stem ~ treatment, groups = variety, type = "connect", data = po2, freckles = TRUE, method = meth, cols = rNcol, ylab = ylabspc, theme = screenThemeSPC(), legend =NULL, xlab = "")
@


%%%%% Figure 3 (3a - 3d 2 x 2 panels)

<< Fig_3, fig.cap =  fn("\\label{physioAll}\\textbf{Physiological traits at 21 days.} Variety T-16 shown in green.  Variety WI-9 shown in red.  Large point is the mean of the category.  Small points are the sample values, jittered horizontally.  Error bars correspond to the 95\\% confidence interval. A: total phenols. GAE = gallic acid equivalents.  B: proline.   C: stem betaxanthins.  D: stem betacyanins."), include = TRUE  >>=

dvp1 <- viewport(x = 0.25, y = 0.75, width = 0.5, height = 0.5) # upper left
dvp2 <- viewport(x = 0.25, y = 0.25, width = 0.5, height = 0.5) # lower left
dvp3 <- viewport(x = 0.75, y = 0.75, width = 0.5, height = 0.5) # upper right
dvp4 <- viewport(x = 0.75, y = 0.25, width = 0.5, height = 0.5) # lower right

grid.newpage()
pushViewport(dvp1)
print(Fig3a, newpage = FALSE)
popViewport()
pushViewport(dvp2)
print(Fig3b, newpage = FALSE)
popViewport()
pushViewport(dvp3)
print(Fig3c, newpage = FALSE)
popViewport()
pushViewport(dvp4)
print(Fig3d, newpage = FALSE)
popViewport()

grid.text("A", x = 0.05, y = 0.95, gp = gpar(fontface = "bold", cex = 1.5))
grid.text("B", x = 0.05, y = 0.45, gp = gpar(fontface = "bold", cex = 1.5))
grid.text("C", x = 0.55, y = 0.95, gp = gpar(fontface = "bold", cex = 1.5))
grid.text("D", x = 0.55, y = 0.45, gp = gpar(fontface = "bold", cex = 1.5))
@

% Setting include = TRUE on this next code chunk will place the figure
% caption in the correct spot as required in a PLOS ONE submission.
% However, include = FALSE here so that the figures will be placed
% by LaTeX.  When include = TRUE, you also need to use endfloats
% in the preamble.
<< Fig_3_Caption, include = FALSE, results = "asis" >>=
cat(figString)
@

Betalain pigment concentrations were measured in the stem tissue.  Fig.~\ref{physioAll} Panels C and D shows the effect of salinity on the production of betaxanthin (yellow) pigments and betacyanin (purple-red) pigments. These pigments are produced in a layer just a few cells thick in the cortical tissue under the stem epidermis, but the extraction process includes all stem tissue (\nameref{S1_Photo}).  As a result, the values reported here are low; the concentration of pigment is diluted by a large amount of interior tissue that does not contain betalain pigments.  Even so, there was a significant variety by treatment interaction for both pigment classes.  Variety WI-9 increased production of betaxanthin and betacyanin pigments under saline stress more than variety T-16.  This result corresponds to our visual observations: WI-9 stems were distinctly more red-purple than T-16 stems, especially so in the saline treatments.  Our analysis of betaxanthin and betacyanin pigments in the leaves revealed that both varieties increased pigment production with saline stress, but there was no variety by treatment interaction in the leaves (\nameref{S4_Fig} and \nameref{S5_Fig}).

%%%%%  Table 2
\vspace{0.5 cm}

<< Table2, include = TRUE, results = "asis" >>=

# run the anovas
aov1 <- tidy(aov(ppmGAE ~ treatment*variety, data = po2))
aov2 <- tidy(aov(proline ~ treatment*variety, data = po2))
aov3 <- tidy(aov(ug.xanth.frsh.stem ~ treatment*variety, data = po2))
aov4 <- tidy(aov(ug.cyan.frsh.stem ~ treatment*variety, data = po2))

# assemble
bigaov <- rbind(aov1, aov2, aov3, aov4)
bigaov$df <- as.integer(bigaov$df)
bigaov <- cbind(
	c("\\textbf{phenols}", rep("", 3), "\\textbf{proline}", rep("", 3), "\\textbf{stem}",
	"\\textbf{betaxanthins}", rep("", 2), "\\textbf{stem}", "\\textbf{betacyanins}", rep("", 2)), bigaov)
names(bigaov)<- c("\\textbf{trait}", "\\textbf{term}", "\\textbf{df}", "\\textbf{SS}",
	"\\textbf{MS}", "\\textbf{F-statistic}", "\\textbf{p-value}")

# format for output
table2 <- xtable(bigaov, label = "Table2", caption = "\\textbf{ANOVA results for physiological traits.}")
align(table2) <- "rrrrrrrr"
digits(table2)[8] <- -2
#digits(table2)[3] <- 0
digits(table2)[4:7] <- 1
print(table2, floating = TRUE, include.rownames = FALSE,
	hline.after = c(-1, 0, 4, 8, 12, 16), booktabs = TRUE,
	sanitize.text.function = function(x){x},
	scalebox = 0.85, table.placement = "H",
	latex.environment = "")
@

\vspace{0.5 cm}

<<  LeafBetIndices >>=
A538LF <- colMeans(betaleaf[c(431:437),-1])
A600LF <- colMeans(betaleaf[c(606:612),-1])
AbetacyaninsLF <- 1.095*(A538LF-A600LF) # using the indices
impuritiesLF <- A538LF - AbetacyaninsLF
A476LF <- colMeans(betaleaf[c(259:265),-1])
AbetaxanthinsLF <- A476LF - impuritiesLF - (AbetacyaninsLF/3.1)

# take into account the assay volumes & unit conversions
cyanLF <- AbetacyaninsLF * 3 * 1.5 * 0.01 * 10^6/1120
xanthLF <- AbetaxanthinsLF * 3 * 1.5 * 0.01 * 10^6/750

# add to master data frame
cyanLF <- data.frame(cyanLF = unlist(cyanLF), spc = names(cyanLF))
po2 <- join(po2, cyanLF, by = "spc", type = "left")
xanthLF <- data.frame(xanthLF = unlist(xanthLF), spc = names(xanthLF))
po2 <- join(po2, xanthLF, by = "spc", type = "left")
@


<<  LeafBetaxanA >>=
po2$ug.xanth.frsh.leaf <- 1000*po2$xanthLF/(po2$BCA.beta.lf.wt.mass.7.1)
@

<<  LeafBetacyanA >>=
po2$ug.cyan.frsh.leaf <- 1000*po2$cyanLF/(po2$BCA.beta.lf.wt.mass.7.1)
@

\subsection*{Saline Stress Reveals Alternative Strategies}

Examining pairwise combinations of variables reveals that the two purslane varieties employ alternative strategies when faced with saline stress (Fig.~\ref{pairs}).  The above-ground fresh mass is a surrogate measure for reproductive fitness.  In the absence of saline stress, variety WI-9 produces slightly smaller plants on average, but there is no significant variation between the two varieties.  In addition, both varieties produce the same amount of proline (Fig.~\ref{pairs}, panel A).   However, when exposed to the 200 mM NaCl treatment, variety T-16 loses less mass but invests more in the production of proline, compared to WI-9 plants.  Comparing the above-ground fresh mass with the concentrations of betacyanin pigments in the stems, one sees an opposite trend (Panel B).  In this case, variety WI-9 exposed to 200 mM NaCl invests more in the synthesis of betacyanins in the stem, compared to the T-16 plants.  These alternative strategies (investments) are even more apparent in Panel C, which compares the concentration of proline directly to the stem betacyanin concentration.  In this case, both varieties are very similar in the absence of NaCl, but under the 200 mM NaCl treatment, they diverge substantially.  Plants of variety T-16 investing relatively more in proline and variety WI-9 invests relatively more in betacyanins.   While both proline and betalain pigments undoubtedly play multiple roles, one interpretation of these data is that variety T-16 employs proline as a primary antioxidant to protect against stress while variety WI-9 employs betacyanins in this role.

<<  pairsData >>=
# Create a dedicated df for the next few analyses
po3 <- po2
po3$trt <- as.factor(ifelse(po3$stake == "red", "H", "L"))
po3$GxE <- as.factor(paste(po3$variety, po3$trt, sep="."))
keep <- c(20, 21, 22, 23, 32, 33, 35, 37, 40, 41, 42, 45, 46, 48) # variables of interest
po4 <- na.omit(po3[keep])
names(po4) <- c("nodes", "branches", "flowers", "dropped lvs", "net dry mass", "above-ground\nfresh mass", "total phenols", "proline", "stem\nbetaxanthins", "stem\nbetacyanins", "total stem\nbetalains", "leaf\nbetaxanthins", "leaf\nbetacyanins", "GxE")
@

%%%%% Figure 4 (Relationships among selected variables)

<<  Fig_4, fig.cap = fn("\\label{pairs}\\textbf{Relationships among selected variables}."), fig.pos = "h", include = TRUE >>=
po5 <- po4[,c(6, 8,  9, 10, 14)]
names(po5) <- c('agfm', 'proline', 'stbx', 'stbc', 'GxE')
#po5$mpc <- c("transparent", "darkgreen","darkgreen","red", "red")[po5$GxE]
#po5$mpch <- c(NA, 21, 1, 24, 2)[po5$GxE]
po5$mpc <- c("darkgreen","darkgreen","red", "red")[po5$GxE]
po5$mpch <- c(21, 1, 24, 2)[po5$GxE]

# 4 viewports with 3 data sets and a legend
# vp's defined in ndc

dvp1 <- viewport(x = 0.25, y = 0.75, width = 0.5, height = 0.5) # upper left
dvp2 <- viewport(x = 0.25, y = 0.25, width = 0.5, height = 0.5) # lower left
dvp3 <- viewport(x = 0.75, y = 0.75, width = 0.5, height = 0.5) # upper right
lvp <- viewport(x = 0.75, y = 0.25, width = 0.5, height = 0.5, just = c("center", "center")) # lower right

leg <- draw.key(key = list(
    text = list(
        lab = c("T-16       0 mM NaCl", "T-16   200 mM NaCl", "WI-9       0 mM NaCl", "WI-9   200 mM NaCl"),
            col = "black"),
    points = list(
        pch = c(1, 21, 2, 24),
        fill = c("darkgreen", "darkgreen", "red", "red"),
        col = c("darkgreen", "darkgreen", "red", "red")
        ),
    columns = 1
    ))

p1 <- xyplot(proline ~ agfm, data = po5, xlab = "above-ground fresh mass (g)",
	ylab = expression(proline/fresh~leaf~(mu*g/g)), col = po5$mpc, pch = po5$mpch, fill = po5$mpc)

p2 <- xyplot(proline ~ stbc, data = po5, xlab = expression(betacyanins/fresh~stem~(mu*g/g)),
	ylab = expression(proline/fresh~leaf~(mu*g/g)), col = po5$mpc, pch = po5$mpch, fill = po5$mpc)

p3 <- xyplot(stbc ~ agfm, data = po5, xlab = "above-ground fresh mass (g)",
	ylab = expression(betacyanins/fresh~stem~(mu*g/g)), col = po5$mpc, pch = po5$mpch, fill = po5$mpc)

grid.newpage()
pushViewport(dvp1)
print(p1, newpage = FALSE)
popViewport()
pushViewport(dvp2)
print(p3, newpage = FALSE)
popViewport()
pushViewport(dvp3)
print(p2, newpage = FALSE)
popViewport()
pushViewport(lvp)
grid.draw(leg)
popViewport()
grid.text("A", x = 0.05, y = 0.95, gp = gpar(fontface = "bold", cex = 1.5))
grid.text("B", x = 0.05, y = 0.45, gp = gpar(fontface = "bold", cex = 1.5))
grid.text("C", x = 0.55, y = 0.95, gp = gpar(fontface = "bold", cex = 1.5))
@

% Setting include = TRUE on this next code chunk will place the figure
% caption in the correct spot as required in a PLOS ONE submission.
% However, include = FALSE here so that the figures will be placed
% by LaTeX.  When include = TRUE, you also need to use endfloats
% in the preamble.
<< Fig_4_Caption, include = FALSE, results = "asis" >>=
cat(figString)
@

The alternative strategies for responding to saline stress are also evident in a principal components analysis (PCA) that considers several variables simultaneously (Fig.~\ref{scores}).  Variables included were node count, flower count, branch count, dropped leaf count, above-ground dry mass, proline concentration, and concentration of betacyanins in the stem.  The inclusion of additional variables, such as the stem betaxanthin concentrations, gave poorer results, suggesting that the chosen variables best represented the underlying biology and chemistry.  Values were scaled to unit variance prior to PCA.  In this plot, excellent separation is observed according to the variety by treament categories.  Principal component 1 largely reflects the separation by variety, while principal component 2 largely reflects the separation by treatment. 

<<  PCA_1 >>=
pcaData <- na.omit(po4[c(1:5, 8, 10)])
pca <- prcomp(pcaData, scale. = TRUE)
#plot(pca, main = "Scree Plot")
@

%%%%% Figure 5 (Score Plot)

<<  Fig_5, fig.cap = fn("\\label{scores}\\textbf{Score plot.}"), fig.pos = "h", include = TRUE >>=

scores <- as.data.frame(pca$x)
scores <- data.frame(scores, GxE = po4$GxE)

xyplot(PC1 ~ PC2, data = scores, xlab = "PC1 (44% of variation)", ylab = "PC2 (33% of variation)",
pch = c(21, 1, 24, 2)[scores$GxE],
fill = c("darkgreen","darkgreen","red", "red")[scores$GxE],
col = c("darkgreen","darkgreen","red", "red")[scores$GxE], asp = 1)
pushViewport(viewport(x = 0.78, y = 0.85, width = 0.5, height = 0.5))
grid.draw(leg)
@

% Setting include = TRUE on this next code chunk will place the figure
% caption in the correct spot as required in a PLOS ONE submission.
% However, include = FALSE here so that the figures will be placed
% by LaTeX.  When include = TRUE, you also need to use endfloats
% in the preamble.
<< Fig_5_Caption, include = FALSE, results = "asis" >>=
cat(figString)
@

\section*{Discussion}

In this study, we have investigated morphological and physiological traits which characterize the response of purslane to saline stress.  Morphological traits, such as the number of reproductive structures and above-ground biomass, serve as proxies for reproductive fitness.  Physiological variables, on the other hand, provide a snapshot of the biochemical changes invoked by the stressor.  We have described substantially different responses to saline stress in the two varieties of purslane that we studied. Such variety by treatment interaction effects are due in part to genetic differences between the varieties, and in part to phenotypic plasticity, the ability of a variety to respond in a flexible way to different environmental conditions.  Phenotypic plasticity may reveal genetic variation which is hidden when the environmental conditions are relatively benign.  However, under appropriate stress, this variation can be uncovered when a particular variety performs better under stress.  Phenotypic plasticity is one of the factors that can drive speciation\cite{Price2003} and can serve as a buffer against extinction in the face of climate change. \cite{Valladares2007,Nicotra2010} Studying the relationships among plastic responses of multiple traits yields a holistic view of purslane's available strategies when responding to saline stress.

In general, plants react to salinity stress using some combination of three cellular mechanisms:  excluding Na\textsuperscript{+} from the tissues (by preventing it from entering the roots or sequestering it in vacuoles), adapting as much as possible to the toxicity of excess Na\textsuperscript{+} in the tissues, and by adjusting to the increased osmotic pressure (which is more generic and can be induced by many molecular species).  Complicating things, the response to Na\textsuperscript{+} is intertwined with the transport and balance of water, K\textsuperscript{+} and Cl\textsuperscript{-} in a poorly understood manner.\cite{Plett2010}  There is also a temporal aspect to the response as described by Munns.\cite{Munns2008} The response of purslane to salt stress which we observed appears to involve the ionic phase identified by Munns in which the concentration of Na\textsuperscript{+} in the plant cells become toxic.  This phase occurs after a shorter acclimation to the osmotic insult, and is characterized by the death of older leaves (dropped leaves in this study) and greatly reduced growth of new tissues (above-ground fresh mass in this study).  At the concentrations and time frame employed in this study, purslane is unable to exclude NaCl from its tissues, consistent with the work of Teixeira and Carvalho\cite{Teixeira2009} who found that Na\textsuperscript{+} concentrations in the tissues increased with increasing salinity of the water, using salinity ranges similar to this work.

Other researchers have reported studies of purslane under saline stress, measuring a variety of traits.\cite{Teixeira2009, Franco2011, Anastacio2013, Yazici2007} However, only a limited number of studies considered the traits explored here.  These studies will be discussed in the next few sections. 

\subsection*{Morphological Traits}

The morphological traits measured in this study serve as proxies for reproductive fitness, as they represents investments in reproductive structures and the tissues that support them.\cite{Hunt2010}  In our experiments, the flower count, node count, and above-ground dry mass  all decreased with increasing salt concentration, and the dropped leaf count increased (Fig.~\ref{morphAll}). Both varieties showed substantial, statistically significant reduction in these fitness proxies under increased salinity, though the extent of the response varied between the two varieties (Table~\ref{Table1}). These results demonstrate that 200 mM NaCl severely stresses purslane without killing it, and suggest that purslane is not strictly a glycophyte nor is it a halophyte (though Al Nasir \emph{et al.} consider purslane to be a halophyte\cite{AlNasir2010}).  The other morphological traits we measured, namely the branch count, the above-ground fresh mass and the ratio of above-ground dry to fresh mass (\nameref{S1_Fig}, \nameref{S2_Fig} and \nameref{S3_Fig}) also demonstrated a significant stress response for both varieties, as well as substantial differences between the varieties.  These observations are consistent with the overall habits and architectures of the two varieties (Fig.~\ref{morphFotos}).  Variety T-16 (domesticated 'tall-green') grows tall, produces many leaves, flowers late, branches late and flowers largely at the apical meristem.  On the other hand, variety WI-9 (wild-type) sprawls, branches frequently and flowers early at most leaf axils.  These differences drive some of the variation observed, but the overall trends we observe also reflect the suppression of flowering in WI-9 and node formation in T-16 by the saline treatment.  Our measures of biomass are consistent with those of Alam \emph{et al.} who have studied a number of purslane accessions with regard to salinity tolerance.\cite{Alam2014b, Alam2015a, Alam2015b}.  They also found that the loss of biomass depended upon the variety, and in general, was quite substantial. Similar trends were found for the number of flowers, leaves and dropped leaves, though the methodological details differed from our study.

\subsection*{Physiological Traits}

Phenolic compounds such as phenolic acids, anthocyanins (not found in purslane), flavonoids and tannins serve multiple roles in plants, including functioning as antioxidants.  We assayed the total phenolic content of purslane using the BCA assay.\cite{Marino2009}  This assay responds to phenolic substances in a similar fashion as the more commonly used Folin-Ciocalteu assay\cite{Waterhouse2002} but is more sensitive, and the color development is stable.  We observed a significant variety by treatment interaction in which the phenolic content of variety WI-9 was nearly inert to increased NaCl, while variety T-16  \emph{decreased} the production under saline treatment (Fig.~\ref{physioAll} Panel A, Table~\ref{Table2}).  The values found here with the BCA assay are in the range of those found by Alam \emph{et al.} for purslane using the Folin-Ciocalteu assay (after correction to a dry mass basis); they also observed that varieties may produce more \emph{or} less total phenols under saline stress.\cite{Alam2015a}

Up-regulation of the amino acid proline in stressed plants has been known for decades (reviewed in \cite{Ashraf2007}; see \cite{Penuelas2013} for recent studies) and its role has generally been understood as an osmoprotectant, one of several small molecules that plants produce for this purpose.   However, the question of 'why proline?' has been clarified with the observation that proline functions as an antioxidant via its secondary amine (absent from all the other amino acids).\cite{Matysik2002}  Proline also plays other intertwined and somewhat poorly understood roles.\cite{Szabados2010} In our experiment, each variety had similar proline concentrations in the absence of saline stress, but dramatically increased concentrations when watered with 200 mM NaCl.  Variety T-16 produces more proline on average (Fig.~\ref{physioAll} Panel B).  In a study of purslane on saline soils in Jordan, Al Nasir \emph{et al.} found that the amount of proline increased under stress in autumn but decreased in the spring.\cite{AlNasir2010} On the other hand, Yazici \emph{et al.}\cite{Yazici2007} and Kafi and Rahimi\cite{Kafi2011} found significant increases in proline in experiments very similar to the one described here.

The role of betalain pigments in plant physiology is less well-studied than that of the anthocyanins, but presumably the two pigment families fulfill similar functions.  Like proline, the betalain pigments are antioxidants.\cite{Kanner2001, Wybraniec2011}  We observed significant variety by treatment interactions for both stem betacyanins and stem betaxanthins (Fig.~\ref{physioAll} Panels C and D). This quantitative analysis is in line with our visual observations: the saline stressed plants are strikingly more red-purple.

These two pigment categories correlate strongly (\nameref{S6_Fig}).  This likely occurs because they share a common biosynthetic intermediate, betalamic acid, and the downstream reactions are spontaneous, not enzyme catalyzed.\cite{Schliemann1999} Thus any betalamic acid produced can react with any available amino acids to give betaxanthins, or with a cyclo-DOPA derivative to give betacyanins, with more or less equal likelihood.

To our knowledge, this is one of the few studies to quantify an increase in betalain pigments with saline stress.  Chang-Quan \emph{et al.}, investigating \emph{Suaeda salsa}, a C-3 halophyte, observed that betalain levels decreased initially under saline stress but then rebounded, and never exceeded their initial levels.\cite{Chang-Quan2007}  Jain and Gould reported that red, but not green, morphs of the coastal succulent \emph{Disphyma australe} increase their betalain content 10-fold in response to salinity.\cite{Jain2015}  The only report on \emph{P. oleracea} is from Alam \emph{et al.}, who briefly noted a visual increase in red-pink pigments with saline stress.\cite{Alam2014b}

\subsection*{Alternative Investments as Adaptive Strategies}

Considering the responses we observed as a whole, it is clear that the two varieties of  purslane that we studied adopt different strategies when investing in resources to cope with saline stress. These alternative responses consist of complex, simultaneous changes in multiple traits, and may represent evidence of a trade-off among multiple alternatives for coping with saline stress.  The most noteworthy changes in traits due to saline stress within the two varieties are summarized in Table~\ref{Table3}.

%%%%% Table 3

\vspace{0.5 cm}

<< Table3, include = TRUE, results = "asis" >>=
# Make table as a data frame so it can be formatted exactly like the others.
DF <- data.frame(
	Trait = c("flower count", "node count", "dropped leaves", "dry mass", "total phenols", "proline", "stem betacyanins"),
	VarietyT16 = c("$\\sim$", "\\texttt{-}", "+", "\\texttt{-}", "\\texttt{-}", "++", "+"),
	VarietyWI9 = c("\\texttt{-{}-}", "$\\sim$", "$\\sim$", "\\texttt{-{}-}", "$\\sim$", "+", "++"))
names(DF) <- c("\\textbf{trait}", "\\textbf{variety\nT-16}", "\\textbf{variety\nWI-9}")

table3 <- xtable(DF, label = "Table3", caption = "\\textbf{Summary of noteworthy changes in traits under saline stress.}")
align(table3) <- "rrrr"
print(table3, floating = TRUE, include.rownames = FALSE,
	hline.after = c(-1, 0, nrow(table3)), booktabs = TRUE,
	sanitize.text.function = function(x){x},
	scalebox = 0.85, table.placement = "H",
	latex.environment = "")
@

<< Table3_Legend, include = TRUE, results = "asis" >>=
tabLeg <- "Key: $\\sim$ largely unchanged; -- decreased; \\texttt{-{}-} strongly decreased; + increased; ++ strongly increased."
cat(tabLeg)
@

\vspace{0.5 cm}

Strategies for the allocation of metabolic resources can be better appreciated if the data are examined holistically instead of considering single traits in isolation.  Overall, variety WI-9 suffers a substantial reduction in flowering and above-ground dry biomass under saline stress, and responds with a large increase in stem betacyanin pigments and a modest increase in proline concentration.  Variety T-16 on the other hand suffers modest decreases in above-ground biomass and node count, an increase in the number of dropped leaves, greatly increased proline and only modestly increased stem betacyanins.  With respect to proline and stem betacyanins, variety T-16 inverts the response of variety WI-9.  This is most apparent in panel C of Fig.~\ref{pairs} and this plot is probably the best illustration of the different strategies employed by these two varieties.  These alternative responses are clearly reflected in the PCA score plot, where one sees a wide separation for each variety by treatment combination (Fig.~\ref{scores}).  This is further evidence that each variety is allocating its morphological and physiological resources in the face of stress according to a different strategy. This may be evidence for a tradeoff between alternatives; perhaps allocation to betalains precludes up-regulation of proline under severe saline stress.  If the experiment were carried out longer, some of these trends might be even more apparent (for instance, at 21 days, T-16 has barely begun to flower). 

Proline, betalain pigments and phenols are metabolites with multiple roles.  Proline is an osmoprotectant, antioxidant and serves other roles as well.\cite{Szabados2010}  Betalains function to attract pollinators, to protect tissues from UV light, and as antioxidants.  Phenolic compounds include many subtypes, each with their own, often poorly known functions, and also possess antioxidant behavior to varying degrees.\cite{Hernandez2009}  Salinity induces responses similar to drought since it reduces the water available to the tissues, and both types of abiotic stress are known to increase reactive oxygen species (ROS) which are critically involved in signalling pathways within plants.\cite{Golldack2014}  Thus ROS are damaging to plant tissues and their concentration must be limited, but they are also necessary as secondary messengers, and complete suppression would be as problematic as too much production.\cite{Ozgur2013}  It was recently reported that \emph{Arabidopsis} responds rapidly to saline stress with a Ca\textsuperscript{2+} signaling wave which induces ROS related genes.\cite{Choi2014} In our experiment, we found that some antioxidants increased (proline, betalains) but the total phenols followed a more complex pattern.  Looking at the activity of enzymes that quench ROS, Yacizi \emph{et al.}, using 2-month old purslane plants, also found a complex pattern of increases and decreases over time.\cite{Yazici2007}  In that study, the levels of proline steadily increased with time and NaCl concentration, but the concentration of malondialdehyde, a measure of membrane damage, showed a delayed response, only increasing at 30 days of treatment and not at 18 days.  Coupled with our results, the observed patterns almost certainly reflect that plants need to maintain a balance and avoid a too 'heavy-handed' antioxidant response by decreasing some antioxidant species while increasing others. T\"{u}rkan and Demiral have noted the need for balance between antioxidant quenching enzymes and small molecule antioxidants in various cellular compartments, as well as the partially overlapping and redundant nature of the system.\cite{Turkan2009} It also reflects the heterogeneity of the 'total phenols', as they are really many compounds which may have widely varying roles and individual concentrations which may increase or decrease under stress.  Similarily, it is impossible to disentagle whether the increased proline concentration is functioning as an osmoprotectant or an antioxidant.  Such an approach is too reductionistic, as proline is clearly capable of both functions.  Betalains have not been identifed as osmoprotectants, but are proven antioxidants.  Betacyanins are also phenolic substances, and presumably measured in the BCA assay.  Since our control and experimental plants were exposed to the same light regime and the plants are self-pollinating, the pigment increase has nothing to do with screening UV light or attracting pollinators.  Thus their antioxidant behavior is probably more relevant in this system.

In this work we have identified two varieties of purslane that employ alternative strategies in allocating resources in the face of saline stress.   This establishes purslane as a suitable model system for the study of saline stress and the molecular basis for differential responses.  We also report for the first time a significant increase in betalain pigments due to saline stress which occurs rapidly in young plants, particularly in the wild type WI-9.  The phenotypic plasticity observed here must of course arise from slightly different genes, which respond differentially to environmental signals to give a better (or worse) adapted proteome for a given stress condition.\cite{Ahuja2010}  The proteome in a given situation represents a suite of enzymes and transcription factors which is ideally adapated to the environmental inputs, and effectively supervises the organism's response to the stressor.  This results in varying concentrations of the final metabolites, some of which we have studied in this work.

\section*{Acknowledgements}

We thank Paulina J. Haight, Matthew A. Kukurugya, Vincent S. Guzzetta, Tanner Miller, Kelly Summers, Courtney Brimmer, Elizabeth Botts and Matthew J. Keinsley for optimization of the assays.

\nolinenumbers

\begin{thebibliography}{10}

% 1
\bibitem{Mitich1997}
Mitich L.
\newblock Common purslane (\emph{{P}ortulaca oleracea}).
\newblock Weed Technology. 1997;11(2):394--397.

%2
\bibitem{Holm1991}
Anonymous.  Purslane.
\newblock In: Holm LG, Plucknett DL, Pancho JV, James PH, editors. The World's
  Worst Weeds: Distribution and Biology. Krieger Publications; 1991. p. 78--83.

% 3
\bibitem{Uddin2014}
Uddin MK, Juraimi AS, Hossain MS, Nahar MAU, Ali ME, Rahman MM.
\newblock Purslane Weed (\emph{{P}ortulaca oleracea}): A Prospective Plant
  Source of Nutrition, Omega-3 Fatty Acid, and Antioxidant Attributes.
\newblock Scientific World Journal. 2014, Article ID 951019, 6 pages, 2014. doi:10.1155/2014/951019.

% 4
\bibitem{Zhou2015}
Zhou YX, Xin HL, Rahman K, Wang SJ, Peng C, Zhang H.
\newblock \emph{Portulaca oleracea} L.: A Review of Phytochemistry and
  Pharmacological Effects.
\newblock {BioMed Research International}. 2015, Article ID 925631, 11 pages, 2015. doi:10.1155/2015/925631.

% 5
\bibitem{Murray2014}
Murray M.
\newblock {omega-3 Polyunsaturated fatty acids and their metabolites as
  inhibitors of mammalian tumorigenesis}.
\newblock Phytochemistry Reviews. 2014;13(1, SI):139--156.

% 6
\bibitem{Riediger2009}
Riediger ND, Othman RA, Suh M, Moghadasian MH.
\newblock A Systemic Review of the Roles of omega-3 Fatty Acids in Health and
  Disease.
\newblock Journal of the American Dietetic Association. 2009;109(4):668--679.

% 7
\bibitem{Simopoulos1992}
Simopoulos A, Norman H, Gillaspy J, Duke J.
\newblock Common Purslane - A Source of omega-3-Fatty-Acids and Antioxidants.
\newblock Journal of the American College of Nutrition. 1992;11(4):374--382.

% 8
\bibitem{Alam2014a}
Alam MA, Juraimi AS, Rafii MY, Hamid AA, Aslani F, Hasan MM, et~al.
\newblock Evaluation of Antioxidant Compounds, Antioxidant Activities, and
  Mineral Composition of 13 Collected Purslane (\emph{{P}ortulaca oleracea}
  {L.}) Accessions.
\newblock Biomed Research International. 2014;2014: 1-10.

% 9
\bibitem{Erkan2012}
Erkan N.
\newblock Antioxidant activity and phenolic compounds of fractions from
  \emph{{P}ortulaca oleracea} {L}.
\newblock Food Chemistry. 2012;133(3):775--781.

% 10
\bibitem{Dkhil2011}
Dkhil MA, Moniem AEA, Al-Quraishy S, Saleh RA.
\newblock Antioxidant effect of purslane (\emph{{P}ortulaca oleracea)} and its
  mechanism of action.
\newblock Journal of Medicinal Plants Research. 2011;5(9):1589--1593.

% 11
\bibitem{Ocampo2012}
Ocampo G, Columbus JT.
\newblock Molecular Phylogenetics, historical biogeography, and chromosome
  number evolution of \emph{{P}ortulaca } (Portulacaceae).
\newblock Molecular Phylogenetics and Evolution. 2012;63:97--112.

% 12
\bibitem{Brockington2011}
Brockington SF, Walker RH, Glover BJ, Soltis PS, Soltis DE.
\newblock {Complex pigment evolution in the {C}aryophyllales}.
\newblock New Phytologist. 2011;190(4):854--864.

% 13
\bibitem{Stintzing2007}
Stintzing FC, Carle R.
\newblock Betalains - emerging prospects for food scientists.
\newblock Trends in Food Science \& Technology. 2007;18(10):514--525.

% 14
\bibitem{Allegra2007}
Allegra M, Tesoriere L, Livrea MA.
\newblock Betanin inhibits the myeloperoxidase/nitrite-induced oxidation of
  human low-density lipoproteins.
\newblock Free Radical Research. 2007;41(3):335--341.

% 15
\bibitem{Zhao2013}
Zhao R, Gao X, Cai Y, Shao X, Jia G, Huang Y, et~al.
\newblock Antitumor activity of \emph{{P}ortulaca oleracea} {L.}
  polysaccharides against cervical carcinoma in vitro and in vivo.
\newblock Carbohydrate Polymers. 2013;96(2):376--383.

% 16
\bibitem{Dong2010}
Dong CX, Hayashi K, Lee JB, Hayashi T.
\newblock Characterization of Structures and Antiviral Effects of
  Polysaccharides from \emph{{P}ortulaca oleracea} {L.}
\newblock Chemical \& Pharmaceutical Bulletin. 2010;58(4):507--510.

% 17
\bibitem{Shen2013}
Shen H, Tang G, Zeng G, Yang Y, Cai X, Li D, et~al.
\newblock Purification and characterization of an antitumor polysaccharide from
  \emph{{P}ortulaca oleracea} {L.}
\newblock Carbohydrate Polymers. 2013;93(2):395--400.

% 18
\bibitem{Esmailzadeh2015}
Esmailzadeh A, Zakizadeh E, Faghihimani E, Gohari M, Jazayeri S.
\newblock The effect of purlsane seeds on glycemic status and lipid profiles of
  persons with type 2 diabetes: A randomized cross-over clinical trial.
\newblock {Journal of Research in Medical Sciences}. {2015};{20}({1}):{47--53}.

% 19
\bibitem{Lee2012b}
Lee AS, Lee YJ, Lee SM, Yoon JJ, Kim JS, Kang DG, et~al.
\newblock An Aqueous Extract of \emph{{P}ortulaca oleracea} Ameliorates
  Diabetic Nephropathy Through Suppression of Renal Fibrosis and Inflammation
  in Diabetic db/db Mice.
\newblock American Journal of Chinese Medicine. 2012;40(3):495--510.

% 20
\bibitem{Lee2012c}
Lee AS, Lee YJ, Lee SM, Yoon JJ, Kim JS, Kang DG, et~al.
\newblock \emph{{P}ortulaca oleracea} Ameliorates Diabetic Vascular
  Inflammation and Endothelial Dysfunction in db/db Mice.
\newblock Evidence-Based Complementary and Alternative Medicine. 2012;p.
  741824.

% 21
\bibitem{Wang2010}
Wang CQ, Yang GQ.
\newblock Betacyanins from \emph{{P}ortulaca oleracea} {L.} ameliorate
  cognition deficits and attenuate oxidative damage induced by D-galactose in
  the brains of senescent mice.
\newblock Phytomedicine. 2010;17(7):527--532.

% 22
\bibitem{Moneim2013}
Moneim AEA, Dkhil MA, Al-Quraishy S.
\newblock The potential role of \emph{{P}ortulaca oleracea} as a
  neuroprotective agent in rotenone-induced neurotoxicity and apoptosis in the
  brain of rats.
\newblock Pesticide Biochemistry and Physiology. 2013;105(3):203--212.

% 23
\bibitem{Fontana2006}
Fontana E, Hoeberechts J, Nicola S, Cros V, Palmegiano GB, Peiretti PG.
\newblock Nitrogen concentration and nitrate/ammonium ratio affect yield and
  change the oxalic acid concentration and fatty acid profile of purslane
  (\emph{{P}ortulaca oleracea} {L.}) grown in a soilless culture system.
\newblock Journal of the Science of Food and Agriculture.
  2006;86(14):2417--2424.

% 24
\bibitem{Mortley2012}
Mortley DG, Oh JH, Johnson DS, Bonsi CK, Hill WA.
\newblock Influence of Harvest Intervals on Growth Responses and Fatty Acid
  Content of Purslane (\emph{{P}ortulaca oleracea}).
\newblock Hortscience. 2012;47(3):437--439.

% 25
\bibitem{Palaniswamy2001a}
Palaniswamy U, McAvoy R, Bible B.
\newblock Omega-3 fatty acid concentration in purslane (\emph{{P}ortulaca
  oleraceae}) is altered by photosynthetic photon flux.
\newblock Journal of the American Society for Horticultural Science.
  2001;126(5):537--543.

% 26
\bibitem{Tiwari2008}
Tiwari KK, Dwivedi S, Mishra S, Srivastava S, Tripathi RD, Singh NK, et~al.
\newblock Phytoremediation efficiency of \emph{{P}ortulaca tuberosa} rox and
  \emph{{P}ortulaca oleracea} {L.} naturally growing in an industrial effluent
  irrigated area in {V}adodra, {G}ujrat, {I}ndia.
\newblock Environmental Monitoring and Assessment. 2008;147(1-3):15--22.

% 27
\bibitem{Kilic2008}
Kili\c{c} CC, Kukul YS, Ana\c{c} D.
\newblock Performance of purslane (\emph{{P}ortulaca oleracea} {L.}) as a
  salt-removing crop.
\newblock Agricultural Water Management. 2008;95(7):854--858.

% 28
\bibitem{Imai2007}
Imai S, Shiraishi A, Gamo K, Watanabe I, Okuhata H, Miyasaka H, et~al.
\newblock Removal of phenolic endocrine disruptors by \emph{{P}ortulaca
  oleracea}.
\newblock Journal of Bioscience and Bioengineering. 2007;103(5):420--426.

% 29
\bibitem{Winter2014}
Winter K, Holtum JAM.
\newblock Facultative crassulacean acid metabolism (CAM) plants: powerful tools
  for unravelling the functional elements of CAM photosynthesis.
\newblock Journal of Experimental Botany. 2014;65(13, SI):3425--3441.

% 30
\bibitem{DAndrea2014}
Mat\'{i}as~D'Andrea R, Santiago~Andreo C, Valeria~Lara M.
\newblock {Deciphering the mechanisms involved in \emph{Portulaca oleracea}
  (C-4) response to drought: metabolic changes including crassulacean acid-like
  metabolism induction and reversal upon re-watering}.
\newblock {Physiologia Plantarium}. {2014};{152}({3}):{414--430}.

% 31
\bibitem{IPCC2014a}
Anonymous.
\newblock Climate Change 2014: Impacts, Adaptation, and Vulnerability (5th
  Asssessment Report).
\newblock Intergovernmental Panel on Climate Change; 2014.
\newblock Available from: \url{http://www.ipcc.ch/report/ar5/wg2/}.

% 32
\bibitem{Corlett2013}
Corlett RT, Westcott DA.
\newblock Will plant movements keep up with climate change?
\newblock Trends in Ecology \& Evolution. 2013;28(8):482--488.

% 33
\bibitem{Franks2014}
Franks SJ, Weber JJ, Aitken SN.
\newblock Evolutionary and plastic responses to climate change in terrestrial
  plant populations.
\newblock Evolutionary Applications. 2014;7(1):123--139.

% 34
\bibitem{Sultan2003}
Sultan SE.
\newblock Phenotypic plasticity in plants: a case study in ecological
  development.
\newblock Evolution and Development. 2003;5(1):25--33.

% 35
\bibitem{Nicotra2010}
Nicotra AB, Atkin OK, Bonser SP, Davidson AM, Finnegan EJ, Mathesius U, et~al.
\newblock Plant phenotypic plasticity in a changing climate.
\newblock Trends in Plant Sciences. 2010;15(12):684--692.

% 36
\bibitem{Ahuja2010}
Ahuja I, de~Vos RCH, Bones AM, Hall RD.
\newblock Plant molecular stress responses face climate change.
\newblock Trends in Plant Science. 2010;15(12):664--674.

% 37
\bibitem{Bates1973}
Bates LS, Waldren RP, Teare ID.
\newblock Rapid Determination of Free Proline for Water-Stress Studies.
\newblock Plant and Soil. 1973;39:205--207.

% 38
\bibitem{Kugler2004}
Kugler F, Stintzing F, Carle R.
\newblock Identification of betalains from petioles of differently colored
  Swiss chard (\emph{{B}eta vulgaris} {L}. ssp cicla {[{L.}]} Alef. Cv. Bright
  lights) by high-performance liquid chromatography-electrospray ionization
  mass spectrometry.
\newblock Journal of Agricultural and Food Chemistry. 2004;52(10):2975--2981.

% 39
\bibitem{Elbe2007}
Elbe JH.
\newblock Betalains.
\newblock In: Current Protocols in Food Analytical Chemistry. John Wiley and
  Sons, Inc; 2007. p. F3.1.1--F3.1.7.

% 40
\bibitem{Marino2009}
Marino DC, Sabino LZL, Jr JA, Ruggiero ADA, Moya HD.
\newblock Analysis of the Polyphenols Content in Medicinal Plants Based on the
  Reduction of {Cu(II)}/Bicinchoninic Complexes.
\newblock Journal of Agricultural and Food Chemistry. 2009;57:11061--11066.

% 41
\bibitem{R3.3}
{R Core Team}. R: A Language and Environment for Statistical Computing.
\newblock Vienna, Austria; 2015.
\newblock Version 3.3.
\newblock Available from: \url{http://www.R-project.org/}.

% 42
\bibitem{Handy2015}
Hanson BA. HandyStuff: Handy R Functions \& Stuff I Use, \& You Might Too!;
  2015.
\newblock R package version 0.6.123.
\newblock Available from: \url{github.com/bryanhanson/HandyStuff}.

% 43
\bibitem{Price2003}
Price T, Qvarnstr\"{o}m A, Irwin D.
\newblock The role of phenotypic plasticity in driving genetic evolution.
\newblock Proceedings of the Royal Society B-Biological Sciences.
  2003;270(1523):1433--1440.

% 44
\bibitem{Valladares2007}
Valladares F, Gianoli E, G\'{o}mez JM.
\newblock Ecological limits to plant phenotypic plasticity.
\newblock New Phytologist. 2007;176(4):749--763.

% 45
\bibitem{Plett2010}
Plett DC, M\o{o}ller IS.
\newblock Na\textsuperscript{+} transport in glycophytic plants: what we know
  and would like to know.
\newblock Plant Cell and Environment. 2010;33(11):612--626.

% 46
\bibitem{Munns2008}
Munns R, Tester M.
\newblock Mechanisms of salinity tolerance.
\newblock Annual Review of Plant Biology. 2008;59:651--681.

% 47
\bibitem{Teixeira2009}
Teixeira M, Carvalho IS.
\newblock Effects of salt stress on purslane (\emph{{P}ortulaca oleracea})
  nutrition.
\newblock Annals of Applied Botany. 2009;154(1):77--86.

% 48
\bibitem{Franco2011}
Franco JA, Cros V, Vicente MJ, Mart\'{i}nez-S\'{a}nchez JJ.
\newblock Effects of salinity on the germination, growth, and nitrate contents
  of purslane (\emph{{P}ortulaca oleracea} {L.}) cultivated under different
  climatic conditions.
\newblock Journal of Horticultural Science \& Biotechnology. 2011;86(1):1--6.

% 49
\bibitem{Anastacio2013}
Anast\'{a}cio A, Carvalho IS.
\newblock Accumulation of fatty acids in purslane grown in hydroponic salt
  stress conditions.
\newblock International Journal of Food Sciences and Nutrition.
  2013;64(2):235--242.

% 50
\bibitem{Yazici2007}
Yazici I, T\"{u}rkan I, Sekmen AH, Demiral T.
\newblock Salinity tolerance of purslane (\emph{{P}ortulaca oleracea} {L.}) is
  achieved by enhanced antioxidative system, lower level of lipid peroxidation
  and proline accumulation.
\newblock Environmental and Experimental Botany. 2007;61(1):49--57.

% 51
\bibitem{Hunt2010}
Hunt J, Hodgson D.
\newblock What is Fitness, and How Do We Measure It?
\newblock In: Evolutionary Behavioral Ecology. Oxford University Press; 2010.
  p. 46--70.

% 52
\bibitem{AlNasir2010}
Al~Nasir F, Batarseh M, Abdel-Ghani AH, Jiries A.
\newblock Free Amino Acids Content in Some Halophytes under Salinity Stress in
  Arid Environment, {J}ordan.
\newblock Clean-Soil Air Water. 2010;38(7):592--600.

% 53
\bibitem{Alam2014b}
Alam MA, Juraimi AS, Rafii MY, Abdul~Hamid A, Aslani F.
\newblock Screening of Purslane (\emph{{P}ortulaca oleracea} {L.}) Accessions
  for High Salt Tolerance.
\newblock The Scientific World Journal. 2014;2014: 1-12.

% 54
\bibitem{Alam2015b}
Alam MA, Juraimi AS, Rafii MY, Hamid AA.
\newblock Effect of Salinity on Biomass Yield and Physiological and Stem-Root
  Anatomical Characteristics of Purslane (\emph{Portulaca oleracea} L.)
  Accessions.
\newblock {BioMed Research International}. 2015;2015: 1-16.

% 55
\bibitem{Alam2015a}
Alam MA, Juraimi AS, Rafii MY, Hamid AA, Aslani F, Alam MZ.
\newblock Effects of salinity and salinity-induced augmented bioactive
  compounds in purslane (\emph{Portulaca oleracea} L.) for possible economical
  use.
\newblock {Food Chemistry}. {2015};{169}:{439--447}.

% 56
\bibitem{Waterhouse2002}
Waterhouse AL.
\newblock Determination of Total Phenolics.
\newblock In: Current Protocols in Food Analytical Chemistry. John Wiley and
  Sons, Inc; 2002. p. I1.1.1--I1.1.8.
  
% 57
\bibitem{Ashraf2007}
Ashraf M, Foolad MR.
\newblock Roles of glycine betaine and proline in improving plant abiotic
  stress resistance.
\newblock Environmental and Experimental Botany. 2007;59(2):206--216.

% 58
\bibitem{Penuelas2013}
Pe\~{n}uelas J, Sardans J, Estiarte M, Ogaya R, Carnicer J, Coll M, et~al.
\newblock Evidence of current impact of climate change on life: a walk from
  genes to the biosphere.
\newblock Global Change Biology. 2013;19(8):2303--2338.

% 59
\bibitem{Matysik2002}
Matyski J, Bhalu B, Mohanty P.
\newblock Molecular mechanisms of quenching of reactive oxygen species by
  proline under stress in plants.
\newblock Current Science. 2002;82:525--532.

% 60
\bibitem{Szabados2010}
Szabados L, Savour\'{e} A.
\newblock Proline: a multifunctional amino acid.
\newblock Trends in Plant Science. 2010;15(2):89--97.

% 61
\bibitem{Kafi2011}
Kafi M, Rahimi Z.
\newblock Effect of salinity and silicon on root characteristics, growth, water
  status, proline content and ion accumulation of purslane (\emph{{P}ortulaca
  oleracea} {L.}).
\newblock Soil Science and Plant Nutrition. 2011;57(2):341--347.

% 62
\bibitem{Kanner2001}
Kanner J, Harel S, Granit R.
\newblock Betalains - A new class of dietary cationized antioxidants.
\newblock Journal of Agricultural and Food Chemistry. 2001;49(11):5178--5185.

% 63
\bibitem{Wybraniec2011}
Wybraniec S, Stalica P, Sp\'{o}rna A, Nemzer B, Pietrzkowski Z, Michalowski T.
\newblock Antioxidant Activity of Betanidin: Electrochemical Study in Aqueous
  Media.
\newblock Journal of Agricultural and Food Chemistry. 2011;59(22):12163--12170.

% 64
\bibitem{Schliemann1999}
Schliemann W, Kobayashi N, Strack D.
\newblock The decisive step in betaxanthin biosynthesis is a spontaneous
  reaction.
\newblock Plant Physiology. 1999;119(4):1217--1232.

% 65
\bibitem{Chang-Quan2007}
Wang CQ, Chen M, Wang BS.
\newblock Betacyanin accumulation in the leaves of C-3 halophyte \emph{Suaeda
  salsa} L. is induced by watering roots with $H_{2}O_{2}$.
\newblock Plant Science. 2007;172(1):1--7.

% 66
\bibitem{Jain2015}
Jain G, Gould KS.
\newblock Functional significance of betalain biosynthesis in leaves of
  \emph{Disphyma australe} under salinity stress.
\newblock Environmental and Experimental Botany. 2015;109:131--140.

% 67
\bibitem{Hernandez2009}
Hern\'{a}ndez I, Alegre L, Van~Breusegem F, Munn\'{e}-Bosch S.
\newblock How relevant are flavonoids as antioxidants in plants?
\newblock Trends in Plant Science. 2009;14(3):125--132.

% 68
\bibitem{Golldack2014}
Golldack D, Li C, Mohan H, Probst N.
\newblock Tolerance to drought and salt stress in plants: unraveling the
  signaling networks.
\newblock Frontiers in Plant Science. 2014;5:1--10.

% 69
\bibitem{Ozgur2013}
Ozgur R, Uzilday B, Sekmen AH, Turkan I.
\newblock Reactive oxygen species regulation and antioxidant defence in
  halophytes.
\newblock Functional Plant Biology. 2013;40(8-9, SI):832--847.

% 70
\bibitem{Choi2014}
Choi WG, Toyota M, Kim SH, Hilleary R, Gilroy S.
\newblock Salt stress-induced Ca\textsuperscript{2+} waves are associated with
  rapid, long-distance root-to-shoot signaling in plants.
\newblock Proceedings of the National Academy of Sciences of the United States
  of America. 2014;111(17):6497--6502.

% 71
\bibitem{Turkan2009}
T\"{u}rkan I, Demiral T.
\newblock Recent developments in understanding salinity tolerance.
\newblock Environmental and Experimental Botany. 2009;67(1):2--9.

\end{thebibliography}

%%%%% The following figures are created in the figures folder,
%%%%% but not included in the paper.

%%%%% Supplementary Photo 1

<< S1_Photo  >>=
dvp1 <- viewport(x = 0.5, y = 0.5, width = 0.5, height = 0.5)
grid.newpage()
pushViewport(dvp1)
grid.raster(readJPEG("photos/StemDetail_SM.jpg"))
@

%%%%%  Supplementary Figure 1

<<  S1_Fig >>=
ylabspc <- "branch count"
KM13b <- rxnNorm(formula = branches7.1 ~ treatment, groups = variety, type = "anova", table = c(0.52, 0.8, rNcex), data = po, freckles = TRUE, method = meth, cols = rNcol, ylab = ylabspc, xlab = "treatment" , ylim = c(-1,18), theme = screenThemeSPC(), legend = c(0.83, 0.5, 1.0))
print(KM13b)
@

%%%%% Supplementary Figure 2

<<  S2_Fig >>=
ylabspc <- "above-ground fresh mass (g)"
KM25bbb <- rxnNorm(formula = netFreshMass ~ treatment, groups = variety, data = po, freckles = TRUE, method = meth, ylab = ylabspc, type = "anova", table = c(0.52, 0.8, rNcex), cols = rNcol, ylim= c(-1.0, 18.0), theme = screenThemeSPC(), legend = c(0.82, 0.48, 1.0))
print(KM25bbb)
@

%%%%% Supplementary Figure 3

<<  S3_Fig >>=
ylabspc <- "dry to fresh mass ratio"
KM25b <- rxnNorm(formula = d2w ~ treatment, groups = variety, data = po, freckles = TRUE, method = meth, ylab = ylabspc, type = "anova", table = c(0.52, 0.8, rNcex), cols = rNcol, ylim= c(-0.01, 0.24), theme = screenThemeSPC(), legend = c(0.25, 0.25, 1.0))
print(KM25b)
@

%%%%% Supplementary Figure 4

<<  S4_Fig  >>=
ylabspc <- expression(betaxanthins/fresh~leaf~(mu*g/g))
KM20b <- rxnNorm(formula = ug.xanth.frsh.leaf ~ treatment, groups = variety, type = "anova", table = c(0.52, 0.8, rNcex), data = po2, freckles = TRUE, method = meth, cols = rNcol, ylab = ylabspc, theme = screenThemeSPC(), legend = c(0.22, 0.6, 1.0))
print(KM20b)
@

%%%%% Supplementary Figure 5

<<  S5_Fig >>=
ylabspc <- expression(betacyanins/fresh~leaf~(mu*g/g))
KM22b <- rxnNorm(formula = ug.cyan.frsh.leaf ~ treatment, groups = variety, type = "anova", table = c(0.52, 0.8, rNcex), data = po2, freckles = TRUE, method = meth, cols = rNcol, ylab = ylabspc, theme = screenThemeSPC(), legend = c(0.22, 0.55, 1.0))
print(KM22b)
@

%%%%% Supplementary Figure 6

<<  S6_Fig >>=
p1 <- xyplot(stbx ~ stbc, data = po5, xlab = expression(betaxanthins/fresh~stem~(mu*g/g)), ylab = expression(betacyanins/fresh~stem~(mu*g/g)), col = po5$mpc, pch = po5$mpch, fill = po5$mpc)
print(p1)
pushViewport(viewport(x = 0.28, y = 0.85, width = 0.5, height = 0.5))
grid.draw(leg)
@

%%%%% Add Supplementary Captions

\section*{Supporting Information}

\subsection*{S1 Photo.}
\label{S1_Photo}
\textbf{Cross section of T-16 stem showing the layer of pigment containing cells.}

\subsection*{S1 Fig.}
\label{S1_Fig}
\textbf{Branch count at 21 days.}  See the Yohui of Fig.~\ref{morphAll} for interpretation.

\subsection*{S2 Fig.}
\label{S2_Fig}
\textbf{Above ground fresh mass at 21 days.}  See the caption of Fig.~\ref{morphAll} for interpretation.

\subsection*{S3 Fig.}
\label{S3_Fig}
\textbf{Ratio of dry to fresh mass at 21 days.}  See the caption of Fig.~\ref{morphAll} for interpretation.


\subsection*{S4 Fig.}
\label{S4_Fig}
\textbf{Leaf betaxanthins at 21 days.}  See the caption of Fig.~\ref{morphAll} for interpretation.

\subsection*{S5 Fig}
\label{S5_Fig}
\textbf{Leaf betacyanins at 21 days.}  See the caption of Fig.~\ref{morphAll} for interpretation.

\subsection*{S6 Fig}
\label{S6_Fig}
\textbf{Comparison of stem betacyanins concentration to stem betaxanthins concentration}. The correlation between these variables, ignoring group membership, is 0.93.

\end{document}

